src/main/java/com/ably/kafka/connect/ChannelSinkTask.java

```diff
@@ -87,48 +81,49 @@ public void start(Map<String, String> settings) {
                 case "default":
                     if (logger.isDebugEnabled()) {
                         logger.debug(
-                            String.format(
-                                "severity: %d, tag: %s, msg: %s, err",
-                                severity, tag, msg, (tr != null) ? tr.getMessage() : "null"
-                            )
+                                String.format(
+                                        "severity: %d, tag: %s, msg: %s, err",
+                                        severity, tag, msg, (tr != null) ? tr.getMessage() : "null"
+                                )
                         );
                     }
             }
         };
 
         try {
             ably = new AblyRealtime(config.clientOptions);
-            channel = ably.channels.get(config.channelName, config.channelOptions);
-        } catch(AblyException e) {
+        } catch (AblyException e) {
             logger.error("error initializing ably client", e);
         }
     }
 
     @Override
     public void put(Collection<SinkRecord> records) {
-        if (channel == null) {
+        if (ably == null) {
             // Put is not retryable, throwing error will indicate this
             throw new ConnectException("ably client is uninitialized");
         }
 
-        for (SinkRecord r : records) {
+        for (SinkRecord record : records) {
```

Comment:
I fully support this rename, but while you're in the area this could be `final`:

```suggestion
        for (final SinkRecord record : records) {
```

Reply:
https://github.com/ably/kafka-connect-ably/pull/40/commits/a784be8caba7b7c1d9f0ae2a14ba03eaa21d90e3